src/messages/MClientReply.h

```diff
@@ -212,7 +213,9 @@ struct InodeStat {
         decode(snap_metadata, p);
       }
       if (struct_v >= 6) {
-        decode(fscrypt, p);
+	bool fscrypt_flag;
+
+	decode(fscrypt_flag, p);
```

Comment:
Maybe need to align the above two lines by adding two white spaces ?

Reply:
They're aligned when you look at the resulting code. I guess the 2 space indent doesn't come out looking right in github patches.

Reply:
It seems you are using the tab instead of space indents from my side.

Reply:
Yeah, I am. Is that a problem?

Reply:
Just for the code mixes the space indents and tabs, via the vim editor the code won't be aligned which is hard to read sometimes.

This is trivial here, just ignore it.

Reply:
I have the same problem with emacs too and Replacing tabs with spaces didn't work for me. So I just enable whitespace-mode (M-x whitespace-mode) and verify my code changes before committing. :)